project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 resolvers += "TNM" at "http://nexus.thenewmotion.com/content/groups/public"
 
-addSbtPlugin("com.newmotion" % "sbt-build-seed" % "4.1.2")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
+addSbtPlugin("com.newmotion" % "sbt-build-seed" % "5.0.4")
```

Comment:
Are you aware that `5.0.4` only cross-builds Scala 2.12 and 2.13 by default?  I think in this case as this library only uses akka and akka still supports 2.11 we can probably cross built to all 3 versions still

Reply:
I was aware of the former but not giving much of a thought to 2.11 support. I'll try to add it back.